### DISCUSSION TITLE
Dedicated server save file access?

### DISCUSSION THREAD

**Discussion by marbel9** [FakeFishGames/Barotrauma#12175] [Code questions and modding support]
I'm currently trying to move a campaign save from my computer to a dedicated server (hosted by Citadel) because my internet connection is no longer good enough for the number of people wanting to. I've had a lot of issues with Citadel so I wasn't very surprised when I had a problem doing this but when I spoke to their support they told me that it is impossible to move/write save files onto a dedicated server as designed by Daedalic.

Is this true? I haven't been able to find this documented anywhere else, and instead have found instances of people [doing what I was hoping for.
](https://steamcommunity.com/app/602960/discussions/0/3818529563386480668/)

I'm sorry if this is off-topic but I wasn't sure where else to put it. Thank you!

**Comment by pavel4win** [comment]
The link you provided is to an article I wrote. I had no problems setting up a dedicated server and transferring saves to it.
I used the timeweb provider. A dedicated server for ubuntu cost me 400 rubles a month.
(vika4win is account of my daughter)

If you are satisfied with the price and the provider, I can set up a barotrauma server for you as friendly help. It won't be hard for me.
For this you will need to buy a VPS and give me access to it.

**Reply by marbel9** [reply]
Hi Pavel, I really appreciate the offer and may take you up. My only concerns are that timeweb servers might be too far away (I am in NZ) and that it might be difficult for an English speaker to use.

But in the mean time, are you aware of any built in locks on server save files?
